### ISSUE TITLE
orca data image follow up works

### ISSUE THREAD

**Issue by yangw1234** [intel/analytics-zoo#279]
## Write API

- [x] refine write dataset api

```python
orca.data.image.write_parquet(format=..., output_path=..., *args, **kwargs) # format list: imagenet, mnist, image_folder, voc, etc.
```

## Read API
- [x] Support reading datasets as tf.data.dataset
- [ ] Support reading datasets as pytorch dataloader
```python
orca.data.image.read_parquet(format='tf_dataset|dataloader', input_path=..., *args, **kwargs)
```
- [ ] Support sharding on files or row group
```
orca.data.image.read_parquet(format='tf_dataset|dataloader', input_path=..., world_num=..., shard_num=..., *args, **kwargs)
```
- [ ] Support automatically sharding on files or row group when detected running in distributed training mode

- [ ] Support common preprocessing (decoding, data augmentation, batching) for different vision applications
- [ ] Support data_creator function in tf1 estimator
